cast/src/lib.rs

```diff
@@ -1274,6 +1274,24 @@ impl SimpleCast {
         Ok(parse_bytes32_string(&buffer)?.to_owned())
     }
 
+    /// Decodes checksummed address from bytes32 value
+    pub fn parse_bytes32_address(s: &str) -> Result<String> {
+        let s = strip_0x(s);
+        if s.len() != 64 {
+            eyre::bail!("string not 32 bytes");
```

Comment:
```suggestion
            eyre::bail!("expected 64 byte hex-string, got {s}");
```

Reply:
Done in commit  https://github.com/foundry-rs/foundry/pull/4746/commits/6985389c8b88f32b21910cfe60f992a97edb18a5